### PR TITLE
ci(helm): update README.md badges

### PR DESCRIPTION
**Pull request by michaelbeaumont**
### Checklist prior to review

- [x] Link to docs PR or issue --
- [x] Link to UI issue or PR --
- [x] Is the [issue worked on linked][1]? --
- [x] The PR does not hardcode values that might break projects that depend on kuma (e.g. "kumahq" as a image registry) --
- [x] The PR will work for both Linux and Windows, system specific functions like `syscall.Mkfifo` have equivalent implementation on the other OS --
- [x] Unit Tests --
- [x] E2E Tests --
- [x] Manual Universal Tests --
- [x] Manual Kubernetes Tests --
- [x] Do you need to update [`UPGRADE.md`](../blob/master/UPGRADE.md)? --
- [x] Does it need to be backported according to the [backporting policy](../blob/master/CONTRIBUTING.md#backporting)? --
- [x] Do you need to explicitly set a [`> Changelog:` entry here](../blob/master/CONTRIBUTING.md#submitting-a-patch) or add a `ci/` label to run fewer/more tests?

[1]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

### CODE REVIEW

File: deployments/charts/kuma/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: kuma
 description: A Helm chart for the Kuma Control Plane
 type: application
-version: 1.8.0
-appVersion: 1.8.0
+version: 0.0.0
```
Comment (lahabana) [inline deployments/charts/kuma/Chart.yaml:5]:
```suggestion
version: 0.0.0 # This is set automatically by tools/releases/helm.sh
```

File: deployments/charts/kuma/Chart.yaml [hunk 2]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
...
+appVersion: 0.0.0
```
Comment (lahabana) [inline deployments/charts/kuma/Chart.yaml:6]:
```suggestion
appVersion: 0.0.0 # This is set automatically by tools/releases/helm.sh
```

**Review by lahabana — APPROVED** [review]
LG I'm not sure if all the tooling will be available in CI to run helm-docs

**Comment by michaelbeaumont** [comment]
@lahabana I've made it so I can install just `helm-docs` in this workflow.

**Comment by lahabana** [comment]
> @lahabana I've made it so I can install just helm-docs in this workflow.
Should you just not bother and add the `make dev/tools` + caching to avoid doing this one off thing?

File: .github/workflows/helm-release.yaml [hunk 3]
```diff
@@ -40,10 +40,12 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - name: Install chart-releaser
+      - name: Install packages
         run:
           curl -L "https://github.com/helm/chart-releaser/releases/download/v${CR_VERSION}/chart-releaser_${CR_VERSION}_linux_amd64.tar.gz"
           | sudo tar xvz --directory /usr/bin cr
+
+          sudo GO_PACKAGES="github.com/norwoodj/helm-docs/cmd/helm-docs" ./mk/dependencies/go-deps.sh $(make dev/tools/dir)
```
Comment (lahabana) [inline .github/workflows/helm-release.yaml:48]: Maybe just use a `GO_PACKAGE_GLOB` less exact but simpler to implem ;)

**Comment by michaelbeaumont** [comment]
> Should you just not bother and add the `make dev/tools` + caching to avoid doing this one off thing?

Yeah you're right

**Pull request merged**